meetings/2020-02-11.md

```diff
@@ -0,0 +1,49 @@
+# Presto TSC Meeting
+
+### 2020-02-20
+##### 9:00 AM PST to 10:30 AM PST
+
+##### Community Calendar
+[Here](https://calendar.google.com/calendar/embed?src=linuxfoundation.org_vrjlva5b0u73ps75fvnv5sasi4%40group.calendar.google.com&ctz=America%2FChicago) is the public calendar for the Presto TSC meetings.
+
+##### How to Join
+TSC meetings are open for all to join.
+
+https://zoom.us/j/762982927
+
+##### Agenda and Attending this meeting
+
+To attend this meeting or propose a topic, edit this page and add your name to the list of attendees below along with your organization and location. Add any agenda item you wish to discuss, please include relevant links.
+
+* This meeting is for the members of the technical steering committee for Presto + wider community.
+* To cover everything, discussion may be time-constrained per topic.
+* Topics that require less discussion should be covered first.
+* To respect meeting size, topics should be relevant to the agenda.
+
+### Agenda
+- FYI PrestoCon + CFP closes Feb 7th: https://events.linuxfoundation.org/prestocon/
+- Uber representative on TSC
+- Code of Conduct Approval: https://github.com/prestodb/tsc/pull/11
```

Comment:
I think this item only requires a reminder to the TSC to vote it on the mailing list.